weave/__init__.py

```diff
@@ -39,4 +40,4 @@
 # def _ipython_display_():
 #     return show(ecosystem())
 
-__version__ = "0.24.1"
+__version__ = version.VERSION
```

Comment:
oh nooo! weave in server branch depended on this! :( :( 
https://github.com/wandb/core/pull/15787/files#r1272966069